tools/nrtstress/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"flag"
+	"math/rand"
+	"time"
+
+	"k8s.io/klog/v2"
+
+	"github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/nrtupdater"
+	"github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/nrtupdater/fake"
+)
+
+func main() {
+	var hostname string
+	var tmPolicy string
+	var interval time.Duration
+	var seed int
+	var dryRun bool
+	flag.StringVar(&hostname, "hosthame", "fake-host-0", "fake host name (not validated)")
+	flag.StringVar(&tmPolicy, "tm-policy", "best-effort", "topology manager policy (not validated)")
```

Comment:
IMHO we should set the default to single-numa-node, in case we would like to use this generator together with the scheduler, we won't extra configuration here.

Reply:
fair point.